### PR TITLE
Feature flu square_path

### PR DESCRIPTION
**Pull request by joachimhgg**
- [x] (347) samples/robotbenchmark/maze_runner/protos/MazeBlock.proto
- [x] (348) samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto
- ##SUPERVISOR (349) samples/robotbenchmark/square_path/protos/SquarePathBenchmark.proto
- [x] (356) samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto
- [x] (357) samples/robotbenchmark/wall_following/protos/LinkedWalls.proto
- [x] (068) projects/samples/robotbenchmark/square_path/worlds/square_path.wbt
- [x] (074) projects/samples/robotbenchmark/wall_following/worlds/wall_following.wbt

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
The world (`projects/samples/robotbenchmark/square_path/worlds/square_path.wbt`) is not converted.

**Comment by joachimhgg** [comment]
> The world (`projects/samples/robotbenchmark/square_path/worlds/square_path.wbt`) is not converted. 
It has been converted at the commit [8d271f7935573a09a879ba8299ec5f25f4089e00](https://github.com/cyberbotics/webots/pull/3651/commits/8d271f7935573a09a879ba8299ec5f25f4089e00)

**Review by lukicdarkoo — DISMISSED** [review]
Some worlds in which the PROTOs are used are not converted. For example:
- `VisualTrackingBenchmark` in `projects/samples/robotbenchmark/visual_tracking/worlds/visual_tracking.wbt`
- `RobotProgrammingBenchmark` in `projects/samples/robotbenchmark/robot_programming/worlds/robot_programming.wbt`

**Comment by joachimhgg** [comment]
> Some worlds in which the PROTOs are used are not converted. For example:
> 
>     * `VisualTrackingBenchmark` in `projects/samples/robotbenchmark/visual_tracking/worlds/visual_tracking.wbt`
> 
>     * `RobotProgrammingBenchmark` in `projects/samples/robotbenchmark/robot_programming/worlds/robot_programming.wbt`

These worlds are depending of other robots. I modified the PR description and I will create other PRs for these.

File: projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # The Pioneer 3-DX robot (Adept MobileRobots) is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
 
 PROTO SquarePathPioneer3dx [
-  field SFVec3f    translation     0 0.0975 0                       # Is `Transform.translation`.
+  field SFVec3f    translation     0 0 0.0975                      # Is `Transform.translation`.
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto:7]: Changes to the PROTO header should also be updated in the corresponding documentation of the robot

File: projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto [hunk 2]
```diff
@@ -721,7 +721,7 @@ PROTO SquarePathPioneer3dx [
                 ]
               }
               Transform {
-                rotation 0 0 1 1.5708
+                rotation -1 0 0 -1.570796
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto:724]:
```suggestion
                rotation 1 0 0 1.570796
```

File: projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # The Pioneer 3-DX robot (Adept MobileRobots) is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto:4]: There appears to be a difference compared to the master branch, something seems to be in front of the distance sensors
a
b

I don't know if it's just a visual difference or if it actually impacts the sensors. Any clue what that might be?

File: projects/samples/robotbenchmark/square_path/worlds/square_path.wbt [hunk 4]
```diff
@@ -8,20 +8,19 @@ WorldInfo {
...
 TexturedBackgroundLight {
   luminosity 0.7
 }
 TexturedBackground {
 }
 DEF PIONEER SquarePathPioneer3dx {
-  translation 0 0.0943896 0
-  rotation 0 1 0 4.7123889
+  translation 0 0 0.09439
+  rotation 0 0 1 0
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/square_path/worlds/square_path.wbt:23]:
```suggestion
```

File: projects/samples/robotbenchmark/square_path/worlds/square_path.wbt [hunk 5]
```diff
@@ -49,4 +48,4 @@ RectangleArena {
     metalness 0
   }
   wallHeight 0.5
-}
+}
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/square_path/worlds/square_path.wbt:51]:
```suggestion
}

```

File: projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto [hunk 6]
```diff
@@ -722,7 +722,7 @@ PROTO WallFollowingPioneer3dx [
                 ]
               }
               Transform {
-                rotation 0 0 1 1.5708
+                rotation -1 0 0 -1.570796
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto:725]:
```suggestion
                rotation 1 0 0 1.570796
```

File: projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto [hunk 7]
```diff
@@ -712,7 +712,7 @@ PROTO WallFollowingPioneer3dx [
           boundingObject DEF BOUNDING_WHEEL Group {
             children [
               Transform {
-                rotation 0 0 1 1.5708
+                rotation -1 0 0 -1.570796
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto:715]:
```suggestion
                rotation 1 0 0 1.570796
```

Comment (ad-daniel) [reply projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto:715]: ditto.

File: projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto [hunk 8]
```diff
@@ -4,8 +4,8 @@
 # The Pioneer 3-DX robot (Adept MobileRobots) is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
 
 PROTO WallFollowingPioneer3dx [
-  field SFVec3f    translation     0 0.0975 0                       # Is `Transform.translation`.
-  field SFRotation rotation        0 0 1 0                          # Is `Transform.rotation`.
+  field SFVec3f    translation     0 0 0.0975                       # Is `Transform.translation`.
+  field SFRotation rotation        0 0 1 0                         # Is `Transform.rotation`.
```
Comment (ad-daniel) [inline projects/samples/robotbenchmark/wall_following/protos/WallFollowingPioneer3dx.proto:8]:
```suggestion
  field SFRotation rotation        0 0 1 0                          # Is `Transform.rotation`.
```

File: projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto (hunk 3 above)
Comment (joachimhgg) [reply projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto:4]: It was just a visual difference (problem with translations). It's fixed.

**Comment by joachimhgg** [comment]
> Looks good, just missing docs and some minor doubts.
> 
> The `wall_following.wbt` world cannot be converted in this PR?

It seems the doc were already converted. I fixed the issue and converted wall_following.wbt.

**Review by ad-daniel — APPROVED** [review]
Thank you, the broken tests are unrelated to this PR

**Pull request merged**